.github/workflows/ci.yml

```diff
@@ -431,6 +431,25 @@ jobs:
       - name: Run dependency check
         run: ./ci/check_job_dependencies.sh
 
+  test-packing:
+    runs-on: ubuntu-latest
+    name: Checks Packing 
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Testing Packing
+        run: |
+          set -eo pipefail
+          cargo package --target-dir test-pack
+
+          # Check that Licenses exist
+          cd test-pack/package
+          cd "$(ls -d */ | head -n 1)"
```

Comment:
What's happening here?

Reply:
This goes to the package folder on line 445

The pack command generates the source and the actual crate file in the test-pack/package folder with the version 
name appended at the end.  Since the version name will change, it needs another way to reference the package source file. This is where line 446 comes in which grabs the first directory in the current working directory and cd's
into it.

From there we check if the Licenses exist.

Reply:
Will make those suggested changes now